packages/launcher-component/src/flows/launcher-client-adapters.ts

```diff
@@ -30,7 +30,7 @@ export function buildDownloadNewAppPayload(app: NewApp) {
     parts.push({
       category: 'backend',
       shared: {
-        runtime: {name: app.backend.runtimePickerValue!.id, version: 'community'}
+        runtime: {name: app.backend.runtimePickerValue!.runtimeId, version: 'community'}
```

Comment:
I guess you have to send the versionId here and at line 21?

Reply:
good catch